src/meta/play-meta.js

```diff
@@ -24,6 +24,8 @@ import {
   DynamicRoutes,
   SimpleLiveChat,
   FlipCardEffect,
+  NavBar,
+  
```

Comment:
@Shivam-Katare There is a major fix went in main on this file.. please take the latest.

Reply:
Took the latest in this file.